pkg/controller/database/mysqlserver/managed.go

```diff
@@ -35,7 +36,7 @@ import (
 
 	"github.com/crossplaneio/stack-azure/apis/database/v1alpha3"
 	azurev1alpha3 "github.com/crossplaneio/stack-azure/apis/v1alpha3"
-	azure "github.com/crossplaneio/stack-azure/pkg/clients"
+	azuresql "github.com/crossplaneio/stack-azure/pkg/clients"
```

Comment:
Is `github.com/crossplaneio/stack-azure/pkg/clients` really just SQL client code now? Should it be in a package like `pkg/clients/sql`?

Reply:
No, it's not. Fixed it.